README.md

```diff
@@ -118,6 +118,9 @@ For a more realistic example please have a look at [examples/kubernetes/configma
 jobs:
   # each job needs a unique name, it's used for logging and as an default label
 - name: "example"
```

Comment:
possible point of discussion: i considered using the job name as the prefix, but 2 things stopped me from that:

1. it would change the existing behavior (`sql_` prefix) in a non-backwards-compatible way. however, since this hasn't hit a 1.0 release, i wasn't sure if that would be ok
2. the job name is already used as a default label